docs/tls/README.md

```diff
@@ -6,7 +6,13 @@ In this document, we will configure Minio servers with TLS certificates for both
 
 * Download Minio server from [here](https://docs.minio.io/docs/minio-quickstart-guide)
 
-## 2. Generate TLS certificate
+## 2. Configure with existing certificates
+
+Assuming that you are already having private and public certificates, you will need to copy them under `certs` in your Minio config directory using the names `private.key` and `public.crt` for key and public certificates respectively.
+
+If the certificate is signed by a certificate authority, `public.crt` should be the concatenation of the server's certificate, any intermediates, and the CA's certificate.
```

Comment:
Maybe "CA's certificate" should be "CA's root certificate", for that extra bit of clarity?